client/.eslintrc.yml

```diff
@@ -36,6 +36,8 @@ rules:
   react/jsx-indent-props:
     - error
     - 2
+  react/function-component-definition:
```

Comment:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/function-component-definition.md
let's configure it for `arrow-functions`